egg-herbie/src/lib.rs

```diff
@@ -254,6 +275,136 @@ pub unsafe extern "C" fn egraph_get_simplest(
     })
 }
 
+fn get_op_str(n: &Math) -> Option<String> {
+    match n {
+        Math::Add(_) => Some(String::from("+")),
+        Math::Sub(_) => Some(String::from("-")),
+        Math::Mul(_) => Some(String::from("*")),
+        Math::Div(_) => Some(String::from("/")),
+        Math::Pow(_) => Some(String::from("pow")),
+        Math::Neg(_) => Some(String::from("neg")),
+        Math::Sqrt(_) => Some(String::from("sqrt")),
+        Math::Fabs(_) => Some(String::from("fabs")),
+        Math::Ceil(_) => Some(String::from("ceil")),
+        Math::Floor(_) => Some(String::from("floor")),
+        Math::Round(_) => Some(String::from("round")),
+        Math::Log(_) => Some(String::from("log")),
+        Math::Cbrt(_) => Some(String::from("cbrt")),
+        Math::Other(s, _) => Some(s.to_string()),
+        _ => None,
```

Comment:
This catch-all is probably bad, because when we use this function we just crash when we see None

Reply:
We also don't need this function- see my other comment about joining recexprs

Reply:
Removed